packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1806,7 +1810,11 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @mustCallSuper
   void handleThumbPressUpdate(Offset localPosition) {
     assert(_debugCheckHasValidScrollPosition());
-    final ScrollPosition position = _currentController!.position;
+    if (_lastDragUpdateOffset == localPosition) {
```

Comment:
This will frequently be called when press and hold. There are benefits to short-circuiting it.